object/mailing_list.go

```diff
@@ -144,15 +144,13 @@ func SyncAllNodeFromGoogleGroup() {
 	}
 }
 
-func (r Reply) AddReplyToMailingList() {
-	targetTopic := GetTopic(r.TopicId)
-	targetNode := GetNode(targetTopic.NodeId)
+func (r Reply) AddReplyToMailingList(title string, targetNode *Node) {
```

Comment:
We'd better transfer `targetTopic` here. We may use some of its member variables in the future.

Reply:
And please resolve the failed checks.